lib/maze/client/appium/bb_client.rb

```diff
@@ -27,16 +27,16 @@ def start_scenario
         def device_capabilities
           config = Maze.config
           capabilities = {
-            'noReset' => true,
+            'appium:noReset' => true,
             'bitbar:options' => {
               # Some capabilities probably belong in the top level
               # of the hash, but BitBar picks them up from here.
               'apiKey' => config.access_key,
               'app' => config.app,
               'findDevice' => false,
-              'testTimeout' => 7200,
-              'newCommandTimeout' => 0
-            }
+              'testTimeout' => 7200
+            },
```

Comment:
I've got some doubts here about moving the capability out of `bitbar:options`, having seen the BitBar can be very particular about where capabilities are put.  Having said that, the behaviour of BitBar at times was very surprising and perhaps this is a good change for ironing that out.

Are we certain that BitBar picks up the capabilities in these locations and that they actually take effect?  If so, I'm happy to go with it.

Reply:
I don't believe this should be a problem, as it's a core appium capability and any custom setup BitBar uses shouldn't effect appium picking it up.